phpBB/phpbb/cache/driver/dummy.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\cache\driver;
+
+/**
+ * ACM Dummy Caching
+ */
+class dummy extends \phpbb\cache\driver\base
```

Comment:
@nickvergessen I don't think git mv is necessary. Moves are autodetected as far as I remember. This difference here is that null.php is not actually removed.